mappings/net/minecraft/block/entity/EndGatewayBlockEntity.mapping

```diff
@@ -42,17 +43,18 @@ CLASS net/minecraft/class_2643 net/minecraft/block/entity/EndGatewayBlockEntity
 	METHOD method_11421 needsCooldownBeforeTeleporting ()Z
 	METHOD method_30276 canTeleport (Lnet/minecraft/class_1297;)Z
 		ARG 0 entity
-	METHOD method_31698 (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;)Z
+	METHOD method_31698 isChunkEmptyAt (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;)Z
 		ARG 0 world
-	METHOD method_31699 (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
+		ARG 1 vec
+	METHOD method_31699 getOrCreateTeleportIsland (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
```

Comment:
I suggest `setupTeleportLocation` or `setupExitPortal`.

Reply:
I think that `getOrCreateTeleportIsland` is fine, as long as the return value is documented.

Reply:
Hmm, this may create a tiny island for the exit portal or just create the exit portal if it's located on a big island. Imo such complex behavior should be coined `setup` than described in detail

Reply:
I think `ExitPortal` is better than `TeleportIsland` because it's clear weather if it's enter or exit.

Reply:
Chose `setupExitPortalLocation` as the method does not actually create the portal itself.